stubs/openpyxl/openpyxl/worksheet/worksheet.pyi

```diff
@@ -24,106 +31,106 @@ class Worksheet(_WorkbookChild):
     PAPERSIZE_A5: str
     ORIENTATION_PORTRAIT: str
     ORIENTATION_LANDSCAPE: str
-    def __init__(self, parent, title: Incomplete | None = ...) -> None: ...
+    def __init__(self, parent: Workbook, title: str | None = ...) -> None: ...
     @property
-    def sheet_view(self): ...
+    def sheet_view(self) -> SheetView: ...
     @property
-    def selected_cell(self): ...
+    def selected_cell(self) -> Cell: ...
     @property
-    def active_cell(self): ...
+    def active_cell(self) -> Cell: ...
     @property
-    def page_breaks(self): ...
+    def page_breaks(self) -> PageBreak: ...
     @property
-    def show_gridlines(self): ...
+    def show_gridlines(self) -> Any: ...
     @property
-    def show_summary_below(self): ...
+    def show_summary_below(self) -> Any: ...
     @property
-    def show_summary_right(self): ...
+    def show_summary_right(self) -> Any: ...
     @property
-    def freeze_panes(self): ...
+    def freeze_panes(self) -> Any: ...
     @freeze_panes.setter
     def freeze_panes(self, topLeftCell: Incomplete | None = ...) -> None: ...
-    def cell(self, row, column, value: Incomplete | None = ...): ...
-    def __getitem__(self, key): ...
-    def __setitem__(self, key, value) -> None: ...
-    def __iter__(self): ...
-    def __delitem__(self, key) -> None: ...
+    def cell(self, row: int, column: int, value: str | None = ...) -> Cell: ...
+    def __getitem__(self, key: str) -> Cell | tuple[Cell]: ...
```

Comment:
- It's also okay to pass in an `int` or a `slice`: https://foss.heptapod.net/openpyxl/openpyxl/-/blob/branch/3.0/openpyxl/worksheet/worksheet.py#L282-288
- `tuple[Cell]` means "A `tuple` of length one, in which the sole member is of type `Cell`. If you meant "A `tuple` of unknown length, in which the members are all of type `Cell`, then you want `tuple[Cell, ...]`

```suggestion
    def __getitem__(self, key: str | int | slice) -> Cell | tuple[Cell, ...]: ...
```